explorer/site-config/anvil-catalog/dev/index/studiesEntityConfig.ts

```diff
@@ -1,26 +1,53 @@
-import { AnVILCatalogStudy } from "../../../../app/apis/catalog/anvil-catalog/common/entities";
-import { buildAnVILCatalogStudies } from "../../../../app/apis/catalog/anvil-catalog/common/utils";
+import {
+  AnVILCatalogStudy,
+  anvilCatalogStudyInputMapper,
+  getDbGapId,
+} from "../../../../app/apis/catalog/anvil-catalog/common/entities";
 import * as Components from "../../../../app/components";
 import {
   ComponentConfig,
   EntityConfig,
   ListConfig,
 } from "../../../../app/config/common/entities";
 import * as ViewBuilder from "../../../../app/viewModelBuilders/anvil-catalog/common/viewModelBuilders";
-import { SOURCE_FIELD_KEY, SOURCE_FIELD_TYPE } from "../../tsv-config";
+import { ANVIL_CATALOG_FILTER_CATEGORY_KEYS } from "../../filter-category-keys";
+import { mainColumn } from "../detail/study/overviewMainColumn";
+import { sideColumn } from "../detail/study/overviewSideColumn";
+import { top } from "../detail/study/top";
 
 /**
  * Entity config object responsible to config anything related to the /explore/studies route.
  */
-export const studiesEntity: EntityConfig<AnVILCatalogStudy> = {
+export const studiesEntityConfig: EntityConfig<AnVILCatalogStudy> = {
   detail: {
+    detailOverviews: ["Overview"],
     staticLoad: true,
-    tabs: [],
-    top: [],
+    tabs: [
+      {
+        label: "Overview",
+        mainColumn: mainColumn,
+        route: "",
+        sideColumn: sideColumn,
+      },
+    ],
+    top: top,
   },
+  getId: getDbGapId,
   label: "Studies",
   list: {
     columns: [
+      {
+        componentConfig: {
+          component: Components.Links,
+          viewBuilder: ViewBuilder.buildStudyName,
+        } as ComponentConfig<typeof Components.Links>,
```

Comment:
We should import the `Link` component see `explorer/app/components/Links/components/Link/link.tsx` into `explorer/app/components/index.tsx` and use this component on line `43`. The view builder `buildStudyName` should only need to return `label` and `url`.